src/schemas/v2/combined.json

```diff
@@ -36814,184 +36814,236 @@
+              "rentalEstimate": {
+                "title": "Rental Estimate",
+                "type": "number"
               },
-              "propertyFullDescription": {
-                "title": "Property description",
-                "type": "string"
-              },
-              "uprn": {
-                "title": "Uprn",
-                "type": "string"
-              },
-              "propertyType": {
-                "title": "Property Type",
-                "type": "string",
-                "enum": [
-                  "Apartment",
-                  "Flat",
-                  "Property",
-                  "Detached House",
-                  "Maisonette",
-                  "Semi-detached House",
-                  "Terraced House",
-                  "Others"
-                ]
-              },
-              "tenure": {
-                "title": "Ownership Type",
-                "description": "What type of ownership is the property?",
-                "type": "string",
-                "enum": [
-                  "Freehold",
-                  "Managed Freehold",
-                  "Commonhold",
-                  "Leasehold",
-                  "Other"
-                ]
+              "estimatedPrice": {
+                "title": "Estimated Price",
+                "description": "Approximation of the property's current market value",
+                "type": "number"
               },
-              "BasicDetails": {
-                "title": "Basic Property Details",
-                "type": "object",
-                "properties": {
-                  "bedrooms": {
-                    "title": "Number of bedrooms",
-                    "type": "integer"
-                  },
-                  "bathrooms": {
-                    "title": "Number of bathrooms",
-                    "type": "integer"
-                  },
-                  "yearBuilt": {
-                    "title": "Built Year",
-                    "type": "string",
-                    "format": "date"
-                  },
-                  "floorArea": {
-                    "title": "Floor area",
-                    "type": "object",
-                    "properties": {
-                      "inSquareMeter": {
-                        "type": "object",
-                        "properties": {
-                          "area": {
-                            "title": "Area in sqaure meters",
-                            "type": "number"
-                          },
-                          "pricePerSquareMeter": {
-                            "title": "Price per square meter",
-                            "type": "number"
-                          }
+              "yield": {
+                "title": "Yield",
+                "description": "Return on investment (ROI) generated by an income-producing asset",
+                "type": "number"
+              }
+            }
+          }
+        },
+        "if": {
```

Comment:
I know if-then-else is valid in draft-7 of JSONSchema but elsewhere in this schema we use `oneOf` and `discriminator` keywords to represent conditional schemas. Yes, arguably less clear but there's something to be said for consistency?

Reply:
resolved with oneOf